### PR TITLE
Update the contents of the ros howto

### PR DESCRIPTION
**Pull request by jleibs**
### What
Copy/paste code sections from example that was already updated.

Preview: https://www.rerun.io/preview/b3515b71b3539b8851ca7ba5bbfa7c09c0f4fdf3/docs/howto/ros2-nav-turtlebot

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/3695) (if applicable)

- [PR Build Summary](https://build.rerun.io/pr/3695)
- [Docs preview](https://rerun.io/preview/38ec512eb936cb83893d6f65078d430a83a38dbd/docs) 
- [Examples preview](https://rerun.io/preview/38ec512eb936cb83893d6f65078d430a83a38dbd/examples) 
- [Recent benchmark results](https://ref.rerun.io/dev/bench/)
- [Wasm size tracking](https://ref.rerun.io/dev/sizes/)

### CODE REVIEW

**Review by emilk — APPROVED** [review]

File: docs/content/howto/ros2-nav-turtlebot.md [hunk 1]
```diff
@@ -415,27 +411,30 @@ if node_data:
...
-        # TODO(jleibs): Remove conversion once
-        # [#883](https://github.com/rerun-io/rerun/issues/883) is closed
-        q = np.array([q[1], q[2], q[3], q[0]])
-        rr.log_transform3d(path,
-            rr.TranslationRotationScale3D([t.x, t.y, t.z], rr.Quaternion(xyzw=q), timeless=timeless)
+        rr.log(
+            path,
+            rr.Translation3D(
+                rr.TranslationAndMat3(
+                    translation=trimesh.transformations.translation_from_matrix(world_from_mesh),
```
Comment (emilk) [inline docs/content/howto/ros2-nav-turtlebot.md:418]: Isn't this just returning `world_from_mesh[3, 0:3]` or `world_from_mesh[0:3, 3]`?

File: docs/content/howto/ros2-nav-turtlebot.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ This timestamp will apply to all subsequent log calls on in this callback (on th
 again.
 
 
-### TF to log_transform3d
+### TF to rr.Transform3D
```
Comment (Wumpf) [inline docs/content/howto/ros2-nav-turtlebot.md:148]: maybe codeticks on these?

File: docs/content/howto/ros2-nav-turtlebot.md (hunk 1 above)
Comment (jleibs) [reply docs/content/howto/ros2-nav-turtlebot.md:418]: ... probably?

File: docs/content/howto/ros2-nav-turtlebot.md [hunk 3]
```diff
@@ -415,27 +411,30 @@ if node_data:
...
-        rr.log_transform3d(path,
-            rr.TranslationRotationScale3D([t.x, t.y, t.z], rr.Quaternion(xyzw=q), timeless=timeless)
+        rr.log(
+            path,
+            rr.Translation3D(
+                rr.TranslationAndMat3(
+                    translation=trimesh.transformations.translation_from_matrix(world_from_mesh),
+                    mat3x3=world_from_mesh[0:3, 0:3],
+                )
+            ),
```
Comment (Wumpf) [inline docs/content/howto/ros2-nav-turtlebot.md:421]: Why not 
```
rr.Translation3D(translation=.., mat3x3=...)
```

**Review by Wumpf — APPROVED** [review]

File: docs/content/howto/ros2-nav-turtlebot.md (hunk 2 above)
Comment (jleibs) [reply docs/content/howto/ros2-nav-turtlebot.md:148]: Ha, you must have missed: https://github.com/rerun-io/rerun/pull/3690

File: docs/content/howto/ros2-nav-turtlebot.md (hunk 3 above)
Comment (jleibs) [reply docs/content/howto/ros2-nav-turtlebot.md:421]: Ahh that didn't exist when someone ported the example, which I copied over from.

**Pull request merged**